.github/workflows/create-test-plan.py

```diff
@@ -533,6 +533,7 @@ def spec_to_job(spec: JobSpec, key: str, trackmem_symbol_names: bool) -> JobDeta
             if spec.xcode:
                 job.xcode_sdk = "macosx"
         case SdlPlatform.Android:
+            fpic = True # Required after updating NDK from 21 to 28
```

Comment:
`fpic` only affects arguments to `cmake` (`-DCMAKE_POSITION_INDEPENDENT_CODE=ON`), not the `sdl3.pc` step.
You need to do:
```py
                        job.cflags.append("-fPIC")
                        job.cxxflags.append("-fPIC")
```

Reply:
Thanks a ton!